modules/social_features/social_landing_page/css/social_landing_page.section.hero_small.css

```diff
@@ -0,0 +1,69 @@
+@media (min-width: 1200px) {
+  .paragraph--section .paragraph--hero-small {
+    margin-left: calc(-100vw / 2 + 1200px / 2);
+    margin-right: calc(-100vw / 2 + 1200px / 2);
+  }
+}
+.paragraph--hero-small .hero-small__bgimage-overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  background: linear-gradient(rgba(0, 0, 0, 0.1) 0%, rgba(34, 34, 34, 0.7) 100%);
+}
+.paragraph--hero-small .cover-wrap {
+  justify-content: center;
+}
+@media (min-width: 600px) {
+  .paragraph--hero-small .cover-wrap {
+    min-height: 175px;
+  }
+}
+.paragraph--hero-small .cover-small {
+  max-width: 620px;
+  width: 100%;
+  padding: 0 7px;
+  margin: 0 auto;
+  text-align: left;
+}
+@media (min-width: 600px) {
+  .paragraph--hero-small .cover-small {
+    position: absolute;
+    bottom: 10px;
+    left: 50%;
+    transform: translateX(-50%);
+  }
+}
+.paragraph--hero-small .cover-small .page-title {
+  width: 100%;
+  text-align: left;
+  padding: 30px 0 10px;
+  font-size: 2rem;
+  max-height: 6.5rem;
```

Comment:
In my case with 6.5 just a little bit of the line under was showing. 6.4 fixes it for me.

Reply:
nope